### ISSUE TITLE
Create a basic foundation app for future feature implementation

### ISSUE THREAD

**Issue by sunn-e** [aamba/Aamba#14]
I will be adding a simple android app with tensorflow lite backend. This will work as backbone. 
task list:
- [x] Basic app
- [ ] Shoutout bounding box text.( added on 14th oct)
- [ ] Add vertical line and output left or right
- [ ] Add horizontal line and output quadrant number. e.g; human(object name)+in+left bottom(quadrant location), dog in right lower corner
- [ ] Add  an option to increase the number of lines/blocks.  
    - [ ] ( 3 X 3)
   - [ ] (2 X 4)
- [ ] Add haptic feedback mode(needs brainstorming as to how it cn be useful- do we need external device or smartphone's vibration motor?)
- [ ] Integrating other features in this module

**Comment by sunn-e** [comment]
#15

**Comment by sunn-e** [comment]
#16

**Comment by sunn-e** [comment]
#17 #18

**Comment by sunn-e** [comment]
#19

**Comment by itsaligadimi** [comment]
Hi @sunn-e, I would like to work on this. and I have a question.
There're some issues that start with "Create an app that..." which I didn't quite understand. is it going to be one app having these features or several apps with separate features?

**Comment by sunn-e** [comment]
Hi @itsaligadimi , I want every app to be standing on its own. I do not want a single app with lot of features. I want to create several apps that can be small and easy to maintain. If you  want, I may create a different repository for you for the issue that you want to solve. 
This is all just to develop an array of small size apps so that even a basic smartphone of third world country can run it as per expectation. 
Am I clear? :)
tldr; separate apps one feature mentioned in the issues under separate repositories.

**Comment by itsaligadimi** [comment]
@sunn-e Yeah, I got it. and thanks for allowing me to do this, I'm glad to participate. And i think yeah it would be better to have different repositories for the apps. A few more questions:

**Comment by itsaligadimi** [comment]
@sunn-e Yeah, I got it. and thanks for allowing me to do this, I'm glad to participate. And I think yeah it would be better to have different repositories for the apps. A few more questions:
- What's the code architecture you use?
- I assume every repository will follow the above checklist, right?

**Comment by sunn-e** [comment]
> What's the code architecture you use?

I'm not a professional android developer. I am not aware of any. But I would like to know more about it if you know.

> I assume every repository will follow the above checklist, right?

Not necessary. You may use this as a template or create your own. Everything is fine. Check list is just to keep track of development at a single place.

Please ping me on gitter if you want a new repository for any new idea. The basic object detector app will stay in this /Aamba repo.

**Comment by sunn-e** [comment]
Adding a feature. Why not give sound output for the bounding box.
Example, if an object detected is laptop, app should give sound output as laptop. This seems much better than adding a vertical line and then telling what the object is and where it is. Instead we can add that as advanced feature later on.
